.ahoy.yml

```diff
@@ -121,6 +121,7 @@ commands:
       ahoy start-ckan-job-workers
       ahoy start-mailmock &
       sleep 5 &&
+      ahoy cli "behave ${*:-test/features}" --tags @smoke && \
       ahoy cli "behave ${*:-test/features}" || \
```

Comment:
do you want to exclude smoke from 2nd run? --tags 'not @smoke' https://behave.readthedocs.io/en/latest/tag_expressions.html

Reply:
We could exclude them, but there's little benefit. They don't take long.